types/validator/index.d.ts

```diff
@@ -1063,7 +1063,7 @@ declare namespace validator {
         pointsForContainingNumber?: number | undefined;
         pointsForContainingSymbol?: number | undefined;
     }
-    function isStrongPassword(str: string, options?: strongPasswordOptions): boolean;
+    function isStrongPassword(str: string, options?: strongPasswordOptions): boolean| number;
```

Comment:
```suggestion
    function isStrongPassword(str: string, options?: strongPasswordOptions): boolean;
    function isStrongPassword(str: string, options: strongPasswordOptions & { returnScore: true): number;
```
should this be something more like above?
https://github.com/validatorjs/validator.js/blob/63b61629187a732c3b3c8d89fe4cacad890cad99/src/lib/isStrongPassword.js#L89